### PR TITLE
fix: fix always success cell init

### PR DESCRIPTION
**Pull request by driftluo**
**What this PR does / why we need it**:

This PR fix always success cell init

**Which issue(s) this PR fixes**:
Fixes #

**Which docs this PR relation**:

Ref #

**Which toolchain this PR adaption**:

No Breaking Change

**Special notes for your reviewer**:

NIL

**CI Description**

| CI Name                                   | Description                                                     |
| ----------------------------------------- | --------------------------------------------------------------- |
| *Chaos CI*                                | Test the liveness and robustness of Axon under terrible network condition    |
| *Cargo Clippy*                            | Run `cargo clippy --all --all-targets --all-features`      |
| *Coverage Test*                           | Get the unit test coverage report                             |
| *E2E Test*                                | Run end-to-end test to check interfaces                         |
| *Code Format*                             | Run `cargo +nightly fmt --all -- --check` and `cargo sort -gwc`     |
| *Web3 Compatible Test*                    | Test the Web3 compatibility of Axon                               |
| *v3 Core Test*                            | Run the compatibility tests provided by Uniswap V3             |
| *OCT 1-5 \| 6-10 \| 11 \| 12-15 \| 16-19* | Run the compatibility tests provided by OpenZeppelin           |

**CI Usage**

> Check the CI you want to run below, and then comment `/run-ci`.

**CI Switch**

- [x] Chaos CI
- [x] Cargo Clippy
- [ ] Coverage Test
- [x] E2E Tests
- [x] Code Format
- [x] Unit Tests
- [x] Web3 Compatible Tests
- [x] OCT 1-5 And 12-15
- [x] OCT 6-10
- [x] OCT 11
- [x] OCT 16-19
- [x] v3 Core Tests

### CODE REVIEW

File: protocol/src/lazy.rs [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-use ckb_types::{core::ScriptHashType, packed, prelude::*};
+use ckb_types::{bytes::Bytes, core::cell::CellMeta, core::ScriptHashType, packed, prelude::*};
 
 use crate::ckb_blake2b_256;
+use crate::traits::BYTE_SHANNONS;
 use crate::types::{Hex, MerkleRoot};
 
+use std::sync::OnceLock;
+
+static CELL: OnceLock<CellMeta> = OnceLock::new();
```
Comment (KaoImin) [inline protocol/src/lazy.rs:10]: Why use `OnceLock` rather than `lazy_static!`.

Comment (KaoImin) [inline protocol/src/lazy.rs:11]: Why use `OnceLock` rather than `lazy_static!`.

Comment (KaoImin) [inline protocol/src/lazy.rs:11]: Why use `OnceLock` rather than `lazy_static!`.

Comment (driftluo) [reply protocol/src/lazy.rs:10]: The trend of the times, lazy static has completed its historical role, there are some types in std, I recommend using std in the future

**Review by chaoticlonghair — DISMISSED** [review]

**Review by KaoImin — DISMISSED** [review]

**Review by Flouse — DISMISSED** [review]

File: protocol/src/lazy.rs [hunk 2]
```diff
@@ -22,3 +27,26 @@ lazy_static::lazy_static! {
             .index(0u32.pack())
             .build();
 }
+
+pub fn always_success_script_meta() -> &'static CellMeta {
+    CELL.get_or_init(|| {
+        let capacity = (32 + 8 + 1 + ALWAYS_SUCCESS.len()) as u64 * BYTE_SHANNONS;
```
Comment (chaoticlonghair) [inline protocol/src/lazy.rs:31]: I prefer to [`packed::CellOutputBuilder::build_exact_capacity(..)`](https://github.com/nervosnetwork/ckb/blob/0a3f9b16a1bd2b928b6c82872cfa98ed7b3accc3/util/types/src/extension/capacity.rs#L59).

But I could understand why you write like this: for compatibility.

Comment (driftluo) [reply protocol/src/lazy.rs:31]: @KaoImin

Comment (KaoImin) [reply protocol/src/lazy.rs:31]: If do this, add a unit test case to check that the `CellOutput` of these two method is the same.

**Review by KaoImin — APPROVED** [review]

**Review by chaoticlonghair — APPROVED** [review]

**Pull request merged**